src/api/account.ts

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { AptosConfig } from "./aptosConfig";
-import { AccountAddress } from "../core";
+import { AccountAddress, PrivateKey, Account as AccountModule } from "../core";
```

Comment:
```suggestion
import { AccountAddress, PrivateKey, Account } from "../core";
```

I'm assuming this snuck in there somehow

Reply:
well Account is the name of the current class, had to use a different name